src/util/taproot.rs

```diff
@@ -743,7 +743,7 @@ impl ControlBlock {
         let tweak = TapTweakHash::from_key_and_tweak(self.internal_key, Some(curr_hash));
         self.internal_key.tweak_add_check(
             secp,
-            output_key,
+            &output_key.into_inner(),
```

Comment:
nit: use as_inner() ?

Reply:
Done, squashed, force pushed. Now part of  https://github.com/rust-bitcoin/rust-bitcoin/pull/728/commits/376f654c889f01644e9086e502f08cec0d1b3714